[RUN-4836] Add AbsoluteSessionTimeout runtime setting (11.4.0) (#10218)
mendix/docs: content/en/docs/refguide/runtime/custom-settings/_index.md

| <a id="SessionTimeout" href="#SessionTimeout">SessionTimeout</a> | Defines after how much time a session becomes invalid (in milliseconds). After that timeout,  a session becomes eligible for removal. The session will not be destroyed until the next time a scheduled task runs to clean up the active sessions. <br> {{% alert color="warning" %}} Sessions can be removed immediately by a query to the runtime after the session becomes eligible for removal. Navigating between pages is not enough to trigger a query to the runtime. To force a query to the runtime, use a microflow. For example, create a microflow that shows the Home page, then configure your app's navigation to call this microflow rather than relying on the navigation to directly show the page itself. This will ensure the runtime is queried and the user session is removed. {{% /alert %}} | 600000 (10 minutes) |

> | <a id="AbsoluteSessionTimeout" href="#AbsoluteSessionTimeout">AbsoluteSessionTimeout</a> | Defines after how much time a session becomes invalid (in milliseconds). After that timeout, a session becomes eligible for removal. As opposed to the `SessionTimeout`, the `AbsoluteSessionTimeout` timeout is not affected by user interactions. The session will be destroyed at requests or until the next time a scheduled task runs to clean up the active sessions. If the setting is not set then absolute session timeout is disabled, this is the default. | |

Reviewer: ```suggestion
| <a id="AbsoluteSessionTimeout" href="#AbsoluteSessionTimeout">AbsoluteSessionTimeout</a> | Defines after how much time a session becomes invalid (in milliseconds). After that timeout, a session becomes eligible for removal. As opposed to the `SessionTimeout`, the `AbsoluteSessionTimeout` timeout is not affected by user interactions. The session will be destroyed at requests and periodically when sessions are cleaned up. If the setting is not set then this behavior is disabled, which is the default. | |
```

> | <a id="AbsoluteSessionTimeout" href="#AbsoluteSessionTimeout">AbsoluteSessionTimeout</a> | Defines after how much time a session becomes invalid (in milliseconds). After that timeout, a session becomes eligible for removal. As opposed to the `SessionTimeout`, the `AbsoluteSessionTimeout` timeout is not affected by user interactions. The session will be destroyed at requests and periodically when sessions are cleaned up. If the setting is not set then this behavior is disabled, which is the default. | Disabled. *Introduced in Mendix 11.4.0.* |

Reviewer 2: @atennapel 
What happens if both `SessionTimeout` and `AbsoluteSessionTimeout` are set?
I guess this is only an issue if `SessionTimeout` is (a lot) less than `AbsoluteSessionTimeout`.

Author: - `SessionTimeout` is always active
- If `AbsoluteSessionTimeout` is enabled then the regular `SessionTimeout` will still work as before
- This means that inactive sessions will still be deleted from the database, even before the `AbsoluteSessionTimeout`

Reviewer 2: I've clarified this in e9d6b9116480ad4ce1df470f493a4a93d769d7ba. Hope it makes sense.

Author: Thanks! Yes, it's correct.
